docs/installingNative.md

```diff
@@ -130,43 +119,48 @@ Now you can install the other dependencies.
   cd PrairieLearn
   ```
 
-!!! note "Setup a venv"
+- Set up a Python virtual environment:
 
-    It is recommended to use a virtual environment for Python dependencies. You can create a virtual environment within PrairieLearn:
-
-    === "Native"
+  === "uv"
 
         ```sh
-        python3.10 -m venv venv
-        source venv/bin/activate
+        uv venv --python 3.10 --seed
+        source .venv/bin/activate
         ```
 
-    === "uv"
+  === "Native"
 
         ```sh
-        uv venv --python 3.10
-        source venv/bin/activate
+        python3.10 -m venv .venv
+        source .venv/bin/activate
         ```
 
+  You can run `deactivate` to exit the virtual environment, and `source .venv/bin/activate` to re-enter it.
```

Comment:
Unrelated, but I currently use `mise` for activating/deactivating my venv anytime I enter/leave the directory. Maybe a future PR can add instructions for using `direnv` or `mise` to do this.

Reply:
Neat! Adding docs for this in a followup PR would be great.